README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
 - RSUs are often considered less preferable to grantees since they remove control over when you owe tax. Options, if granted with an exercise price equal to the fair market value of the stock, are not taxed until exercise, an event under the control of the optionee. If a company awards you an RSU or restricted stock award which vests over time, you will be taxed on the vesting schedule. You have been put on “autopilot” with respect to the timing of the tax event. This can be a really bad thing if, on the date of vesting, the shares are worth a lot and consequently you owe a lot of tax.
-- ☝️ **You don't want to confuse “restricted stock units” with “restricted stock,”** an entirely different thing (described next).
-
+- ☝️ **You don’t want to confuse “restricted stock units” with “restricted stock,”** an entirely different thing (described soon).
 
-### Other Equity Concepts
+### Less common types of equity
 
-These are a few different types of equity awards and topics that are less common, but we mention for completeness.
-
-- **“Phantom equity”** is a type of compensation award that references equity, but does not entitle the recipient to actual equity in the business. These awards come under a variety of different monikers, but the key to understanding them is knowing that they are really just cash bonus plans, where the cash amounts are determined by reference to a company's stock. Phantom equity can have significant value, but may be perceived as less valuable by workers because of the contractual nature of the promises. Phantom equity plans can be set up as purely discretionary bonus plans, which is less attractive than owning a piece of something. Two examples of phantom equity are phantom stock and stock appreciation rights:
-	- A [**phantom stock**](http://www.investopedia.com/articles/stocks/12/introduction-phantom-stock.asp) award entitles you to a payment equal to the value of a share of the company's stock, upon the occurrence of certain events.
+- **Phantom equity** is a type of compensation award that references equity, but does not entitle the recipient to actual equity in the business. These awards come under a variety of different monikers, but the key to understanding them is knowing that they are really just cash bonus plans, where the cash amounts are determined by reference to a company’s stock. Phantom equity can have significant value, but may be perceived as less valuable by workers because of the contractual nature of the promises. Phantom equity plans can be set up as purely discretionary bonus plans, which is less attractive than owning a piece of something. Two examples of phantom equity are phantom stock and stock appreciation rights:
```

Comment:
Who gets this, generally? How rare are these plans?

Reply:
🚧